app/models/answer.rb

```diff
@@ -0,0 +1,41 @@
+class Answer < ActiveRecord::Base
+  has_many :comments, as: :commentable
+  has_many :votes, as: :votable
+  belongs_to :question, touch: true
+  belongs_to :responder, class_name: 'User'
+  delegate :username, to: :responder, allow_nil: true
+
+  def editable_by? user
+    responder == user
+  end
+
+  def my_comments
+    self.comments
+  end
+
+  def vote_count
+    self.votes.sum(:value)
+  end
+
+  def voted
+    self.votes.pluck(:voter_id)
+  end
+
+  def recent_comments
+    self.comments.order('created_at ASC')
```

Comment:
This is a potentially misleading name. Recent comments suggests a restriction. This is just all comments oldest first.